### PR TITLE
Bug/webintents

### PR DESCRIPTION
**Pull request by matiu**
- rm unused webintents
 - remove ignoreMobilePause
 - refactor protocol handlers
 - add devel protocol handler in browser
 - add 'message' protocol handler.
 - rm LegacyImport
 - fix protocol handler on cold app starts
 - refactor wallet selectors in buy/sell controllers
 - adds status message "verifying wallet integrity" when checking key derivation (only on mobile)

 - rm old animationService (to be replaced by ionic native animations)

Fixes https://github.com/bitpay/copay/issues/4277

### CODE REVIEW

File: src/js/services/animationService.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ angular.module('copayApp.services').factory('animationService', function(platfor
   };
 
   function cleanUpLater(e, e2) {
+console.log('[animationService.js.62:cleanUpLater:]'); //TODO
```
Comment (JDonadio) [inline src/js/services/animationService.js]: Remove console log

File: src/js/routes.js [hunk 2]
```diff
@@ -640,17 +624,22 @@ angular.module('copayApp').config(function(historicLogProvider, $provide, $logPr
...
       }
-
-      if (!animationService.transitionAnimated(fromState, toState)) {
-        event.preventDefault();
-        // Time for the backpane to render
-        setTimeout(function() {
-          $state.transitionTo(toState);
-        }, 50);
-      }
+//      event.preventDefault();
```
Comment (JDonadio) [inline src/js/routes.js]: You miss comments here

File: src/js/controllers/uri.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+angular.module('copayApp.controllers').controller('uriController',
+  function($rootScope, $stateParams, $log, openURLService) {
```
Comment (JDonadio) [inline src/js/controllers/uri.js]: $rootScope is not used

File: public/views/uri.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+
+<div class="row columns p20" ng-controller="uriController">
+  <div class="text-center">
+    <logo width="146"></logo>
+    <div class="text-white" ng-include="'views/includes/version.html'"></div>
+  </div>
+  <h3 class="text-center">
+    Please wait to be redirected...
```
Comment (JDonadio) [inline public/views/uri.html]: Add translate

**Comment by matiu** [comment]
updated and ready for review.

Thanks @cmgustavo for the PR.

**Comment by cmgustavo** [comment]
Great job!! 

ACK. Tested on iOS.

**Pull request merged**